aggregator/src/aggregator/collection_job_driver.rs

```diff
@@ -456,10 +450,10 @@ impl CollectionJobDriverMetrics {
             .u64_counter("janus_collection_jobs_finished")
             .with_description("Count of finished collection jobs.")
             .init();
-        jobs_finished_counter.add(&Context::current(), 0, &[]);
+        jobs_finished_counter.add(0, &[]);
 
         let http_request_duration_histogram = meter
-            .f64_histogram("janus_http_request_duration_seconds")
+            .f64_histogram("janus_http_request_duration")
```

Comment:
Was changing the histogram name intentional?

Reply:
Yes, intentional -- see response to other comment.